bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_generic.py

```diff
@@ -44,3 +44,8 @@ def pythonizegeneric(klass, name):
     AddPrettyPrintingPyz(klass)
 
     return True
+
+import cppyy
+cppyy.gbl.DeclareGenericCallable = GetCppCallableClass("Generic")
```

Comment:
I would not put these in the global namespace, what about having a submodule for them, e.g. : 
`from ROOT.callables import XX`

Reply:
Most people simply write `import ROOT`. The, you would "force" them to write

```python
import ROOT
ROOT.callable.DeclareCppCallable(["int"], "float")
def f(x):
    return 2.0 * x
```

Reply:
I'm not really in favor of this.